src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.resx

```diff
@@ -162,4 +162,7 @@
   <data name="CmdPackageDirectory" xml:space="preserve">
     <value>PACKAGE_DIRECTORY</value>
   </data>
+  <data name="CmdDGFileIOException" xml:space="preserve">
+    <value>Unable to get a temp path for a dependency graph file for project '{0}'. Cannot add package reference.</value>
```

Comment:
I am not sure this is a really useful error message. Do users usually know wht the dependency graph file is? Shold we say "Unable to generate temporary file. Clean your temporary folder and try again"? Or something along these lines.